### PR TITLE
Feat/rc5

### PR DESCRIPTION
**Pull request by binarious**
@olastor Please do an intensive testing here, too. https://binarious.github.io/

### CODE REVIEW

**Comment by binarious** [comment]
Page seems to be defect. 
```
"Template parse errors:
Can't bind to 'detailed' since it isn't a known property of 'online'. ("/span>
      <span *ngIf="title" class="fill">{{ title }}</span>
      <online *ngIf="isLoggedIn()" [ERROR ->][detailed]="false" title="See who is online"></online>
    </md-toolbar>

"): e@54:35"
```

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ before_install:
 - sh -e /etc/init.d/xvfb start
 before_script:
 - npm install
+- sed -i -- 's/common_1.FORM_DIRECTIVES,//g' node_modules/ng2-charts/components/charts/charts.js
```
Comment (binarious) [inline .travis.yml]: Reason: https://github.com/valor-software/ng2-charts/issues/352

**Comment by olastor** [comment]
I think some files contain deprecated syntax from Angular 1 (`#loginForm="ngForm"` or `#mail="ngModel"`See [here](https://docs.google.com/document/d/1RIezQqE4aEhBRmArIAS1mRIZtWFf6JxN_7B4meyWK0Y/preview))
bildschirmfoto_2016-08-26_00-02-05



I am not very familiar with this...

**Comment by binarious** [comment]
That message appears because of `ng2-charts`. Try running:
```
sed -i -- 's/common_1.FORM_DIRECTIVES,//g' node_modules/ng2-charts/components/charts/charts.js
```
in your project folder. I added this to Travis as well. Issue at ng2-charts: https://github.com/valor-software/ng2-charts/issues/352

**Comment by olastor** [comment]
npm test gives me

`ERROR [config]: Invalid config file!
  SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode`

**Comment by binarious** [comment]
Great, that's a good hint! Will fix this later.

**Comment by binarious** [comment]
Found the problem: https://github.com/angular/angular/issues/10618

**Comment by binarious** [comment]
@olastor Please delete all of your local caches for binarious.github.io and retry testing: https://binarious.github.io/
It works for me now.

**Comment by olastor** [comment]
Seems to work!

**Comment by binarious** [comment]
Yes, this WebSocket error was thrown before. Need to dig into that later. I'll wait for Travis and merge after success.

**Pull request merged**